### ISSUE TITLE
Impossible to use cli commands in CI as they output results in non-conformant format

### ISSUE THREAD

**Issue by in4margaret** [microsoft/botframework-cli#537]
When running cli comands botframework cli output is all over the place. 
For example `luis:train:show` if successful outputs luis json response trailed with text message "Training status successfully output to console" which makes it hard to parse properly. 
If not successful the same commands outputs error in plain text, that makes tackling the error gracefully unnecessary complex in CI environments.  

I don't see the way to disable redundant logs using https://github.com/oclif/command/blob/master/src/command.ts . 

It would be great to generalize output in some way.

**Comment by munozemilio** [comment]
@scheyal we should spec out a standard output for all commands in the CLI to be CI compliant

**Comment by scheyal** [comment]
See PR #580 for CLI output guide

**Comment by munozemilio** [comment]
Hello @in4margaret the message has been removed and a flag has been added to other commands to transform the output to json format. Please let us know if there is any other command you are having issues with.
